lint/libs/lint-checks/src/main/java/com/android/tools/lint/checks/OverdrawDetector.java

```diff
@@ -507,32 +508,29 @@ public OverdrawVisitor(String name, PsiClass cls) {
         public boolean visitClass(UClass node) {
             // Don't go into inner classes
             return mCls.equals(node.getPsi()) || super.visitClass(node);
-
```

Comment:
Looks like we changed the semantics here.
Before this change, any `R.layout.<something>` resource identifier was checked, and now we check only `setContentView()` call arguments. What if some helper (library) method is used instead of `setContentView()`?